### PR TITLE
Features/ui update

### PR DESCRIPTION
**Pull request by ikbensiep**
Smoothing out the styles

### CODE REVIEW

**Comment by evert** [comment]
Looking good!

File: assets/css/main.css [hunk 1]
```diff
@@ -161,13 +228,13 @@ nav .alternate h3 {
 nav .alternate a {
   font-size: 90%;
   font-weight: 700;
-  background: #555;
+  background: #607D8B;
 }
 nav .alternate .type-csv {
-  background: #057212;
+  background: #4CAF50;
```
Comment (evert) [inline assets/css/main.css]: Thanks for keeping the green and orange, they were indeed intentional =)

**Comment by evert** [comment]
Numbers are nearly illegible on my screen. The blue on black kinda hurts. Maybe worth looking for another syntax file

**Comment by ikbensiep** [comment]
> Numbers are nearly illegible on my screen. The blue on black kinda hurts. Maybe worth looking for another syntax file

On my screen numbers are orange o_0

Anyway, how do you like Solarized Dark? -> https://highlightjs.org/static/demo/

**Pull request merged**